diff -aurN tap-plugins-0.3.0.orig/Makefile tap-plugins-0.3.0/Makefile
--- tap-plugins-0.3.0.orig/Makefile	2004-02-04 15:58:30.000000000 +0100
+++ tap-plugins-0.3.0/Makefile	2004-02-08 19:14:15.078346312 +0100
@@ -23,8 +23,8 @@
 # Change this if you want to install somewhere else. In particularly
 # you may wish to remove the middle "local/" part of the path.
 
-INSTALL_PLUGINS_DIR	=	/usr/local/lib/ladspa/
-INSTALL_LRDF_DIR	=	/usr/local/share/ladspa/rdf/
+INSTALL_PLUGINS_DIR	=	$(DESTDIR)/usr/lib/ladspa/
+INSTALL_LRDF_DIR	=	$(DESTDIR)/usr/share/ladspa/rdf/
 
 # NO EDITING below this line is required
 # if all you want to do is install and use the plugins.
@@ -49,31 +49,31 @@
 # RULES TO BUILD PLUGINS FROM C CODE
 
 tap_tremolo.so: tap_tremolo.c tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_tremolo.c -o tap_tremolo.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_tremolo.c -o tap_tremolo.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_tremolo.so tap_tremolo.o
 
 tap_eq.so: tap_eq.c tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_eq.c -o tap_eq.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_eq.c -o tap_eq.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_eq.so tap_eq.o
 
 tap_echo.so: tap_echo.c tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_echo.c -o tap_echo.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_echo.c -o tap_echo.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_echo.so tap_echo.o
 
 tap_reverb.so: tap_reverb.c tap_reverb.h tap_reverb_presets.h tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_reverb.c -o tap_reverb.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_reverb.c -o tap_reverb.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_reverb.so tap_reverb.o
 
 tap_limiter.so: tap_limiter.c tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_limiter.c -o tap_limiter.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_limiter.c -o tap_limiter.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_limiter.so tap_limiter.o
 
 tap_autopan.so: tap_autopan.c tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_autopan.c -o tap_autopan.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_autopan.c -o tap_autopan.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_autopan.so tap_autopan.o
 
 tap_deesser.so: tap_deesser.c tap_utils.h ladspa.h
-	$(CC) -I. -O3 -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_deesser.c -o tap_deesser.o
+	$(CC) -I. $(OPTFLAGS) -Wall -fomit-frame-pointer -fstrength-reduce -funroll-loops -ffast-math -c -fPIC -DPIC tap_deesser.c -o tap_deesser.o
 	$(CC) -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt -o tap_deesser.so tap_deesser.o
 
 
