--- tap-plugins-1.0.1/Makefile.orig	2019-01-20 13:10:05.000000000 +0100
+++ tap-plugins-1.0.1/Makefile	2022-01-27 20:04:57.153315247 +0100
@@ -23,8 +23,8 @@
 # Change this if you want to install somewhere else. In particular
 # you may wish to remove the middle "local/" part of the path.
 
-INSTALL_PLUGINS_DIR	=	/usr/local/lib/ladspa/
-INSTALL_LRDF_DIR	=	/usr/local/share/ladspa/rdf/
+INSTALL_PLUGINS_DIR	=	/usr/lib/ladspa/
+INSTALL_LRDF_DIR	=	/usr/share/ladspa/rdf/
 
 # NO EDITING below this line is required
 # if all you want to do is install and use the plugins.
@@ -37,11 +37,11 @@ INSTALL_LRDF_DIR	=	/usr/local/share/lads
 OS := $(shell uname -s)
 
 CC      = gcc
-CFLAGS  = -I. -O3 -Wall -fomit-frame-pointer -funroll-loops -ffast-math -c -fPIC -DPIC
+CFLAGS  += -I. -Wall -fomit-frame-pointer -funroll-loops -ffast-math -c -fPIC -DPIC
 ifeq ($(OS),Darwin)
-LDFLAGS = -nostartfiles -shared -Wl,-install_name,symbolic -lc -lm
+LDFLAGS += -nostartfiles -shared -Wl,-install_name,symbolic -lc -lm
 else
-LDFLAGS = -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt
+LDFLAGS += -nostartfiles -shared -Wl,-Bsymbolic -lc -lm -lrt
 endif
 MODULES = $(wildcard *.c)
 
@@ -62,11 +62,11 @@ tap_dynamics_st.o: tap_dynamics_presets.
 # OTHER TARGETS
 
 install: all
-	-mkdir -p          $(INSTALL_PLUGINS_DIR)
-	cp *.so            $(INSTALL_PLUGINS_DIR)
-	-mkdir -p          $(INSTALL_LRDF_DIR)
-	cp tap-plugins.rdf $(INSTALL_LRDF_DIR)
-	cp tap_reverb.rdf  $(INSTALL_LRDF_DIR)
+	-mkdir -p          $(DESTDIR)$(INSTALL_PLUGINS_DIR)
+	cp *.so            $(DESTDIR)$(INSTALL_PLUGINS_DIR)
+	-mkdir -p          $(DESTDIR)$(INSTALL_LRDF_DIR)
+	cp tap-plugins.rdf $(DESTDIR)$(INSTALL_LRDF_DIR)
+	cp tap_reverb.rdf  $(DESTDIR)$(INSTALL_LRDF_DIR)
 
 clean:
 	-rm -f *.so *.o *~
